ush/nems_configure.sh

```diff
@@ -47,6 +47,11 @@ CHMPETS=${CHMPETS:-${ATMPETS}}
 
 USE_MOMMESH=${USE_MOMMESH:-"true"}
 
+if [[ $OCNRES = "100" ]];  then 
```

Comment:
This would be better off in CICE_postdet()

Reply:
While it is an ice related variable, this is only in nems.configure, so why wouldn't having it in the file that is related to nems.configure file be more appropriate?

Reply:
Actually this isn't ice related, this is in the  "ALLCOMP_attributes" section so it's more all components/cmeps related, so if it isn't allowed here, where is a not component specific CMEPS or general section that it would be better suited for?

Reply:
Just leave it here. I'd like to get any setting of variables out of these namelist generator scripts, but I don't think we've been that consistent anyway.